guide/src/main/asciidoc/problems/resourceNotFound.adoc

```diff
@@ -23,10 +23,10 @@ Content-Type: application/problem+json
 }
 ```
 
-Either the path parameters don't resolve to an existing resource. Look for the more details in the `issues` property.
+Either a path parameter or an optional child resource path don't resolve to an existing resource. Look for the more details in the `issues` property.
 
 ```
-GET /enterprises/0206731645
+GET /enterprises/0206731645/employees/12345678901
```

Comment:
To clarify #202, can we add additional examples?

`GET /enterprises/0206731645/employees/12345678901` where the enterprise is found, but the employee is not.
I guess in that case we would only put the employeeId in in/name/value, and not the enterpriseNumber?

And `GET /enterprises/0206731645/employees/12345678901/profilePicture` where enterprise and employee are found, but the employee does not have a profilePicture.

To not make the page too verbose, maybe the 3 examples can be represented in tabs.